webkit2/jscore.jsc-exception.lisp

```diff
@@ -64,3 +64,13 @@
 (defcfun "jsc_exception_report" :string
   (exception (g-object jsc-exception)))
 (export 'jsc-exception-report)
+
+(define-condition jsc-exception-condition (error)
+  ((exception :initform (error "There should be a JSCException.")
```

Comment:
The error is a bit confusing, because we could believe that it's _about_ a JS error :)
Maybe "Required argument..." instead.

Reply:
Fixed in 9d7bdf78a785a26a9b26d0e9f0cd93c16fcd84d9.